package-lock.json

```diff
@@ -29,7 +29,7 @@
         "provinces": "^1.11.0",
         "regenerator-runtime": "^0.13.7",
         "register-service-worker": "^1.7.1",
-        "sbc-common-components": "2.4.38",
+        "sbc-common-components": "2.4.46",
```

Comment:
I had a look at all the possible breaking changes from 38 -> 46 according to this:
https://github.com/bcgov/sbc-common-components/releases

It all looks fine :)